proxy.live.php

```diff
@@ -31,7 +31,14 @@
 unset($body['proxyURL']);
 $request = new \CF\API\Request($method, $path, $parameters, $body);
 
-$response = $requestRouter->route($request);
+$isCSRFTokenValid = (($request->getMethod() === 'GET') ? true : \CF\SecurityUtil::csrfTokenValidate($cpanelAPI->getHostAPIKey(), $cpanelAPI->getUserId(), $request->getBody()['cfCSRFToken']));
+unset($body['cfCSRFToken']);
+
+if ($isCSRFTokenValid) {
+    $response = $requestRouter->route($request);
+} else {
+    $response = $apiRouter->getAPIClient()->createAPIError('CSRF Token not valid.');
```

Comment:
`$apiRouter` doesn't exist anymore.  Just have the response be the string "CSRF Token not valid, please refresh the page.".  90% of the times this happens it will be malicious so its okay if its not pretty.